src/gtk/toga_gtk/widgets/combobox.py

```diff
@@ -0,0 +1,50 @@
+from gi.repository import Gtk
+from travertino.size import at_least
+
+from .base import Widget
+
+
+class ComboBox(Widget):
+
+    def create(self):
+        self.native = Gtk.ComboBoxText.new_with_entry()
+        self.native.interface = self.interface
+        self.native.connect('changed', self._on_change)
+        self.native.connect('show', lambda event: self.rehint())
```

Comment:
This seems a little odd - or, at least, possibly something where there is a need for a fix is higher up in the drawing process. Is the issue here that the widget reports an incorrect size until it is actually displayed?

Reply:
Honestly, I just saw this being done in a bunch of GTK widgets and figured it was important. The [button widget does it](https://github.com/pybee/toga/blob/6b9d8050e3e57940e288a5afa0a63e267bcd6631/src/gtk/toga_gtk/widgets/button.py#L12) and the [numberinput widget calls rehint in create](https://github.com/pybee/toga/blob/6b9d8050e3e57940e288a5afa0a63e267bcd6631/src/gtk/toga_gtk/widgets/numberinput.py#L22). After removing this line, I get the same functionality. This might have solved an issue that no longer exists?